package/MDAnalysis/topology/guessers.py

```diff
@@ -225,6 +226,16 @@ def guess_atom_element(atomname):
         return no_symbols
 
 
+def guess_atom_element_from_mass(mass_guess, tol=0.1):
+    for element, mass in tables.masses.items():
```

Comment:
Doing this for every element is going to end up being pretty tedious & expensive. Is there a way we could just store a "rounded to the given precision" dictionary somewhere and just round the input value?

Reply:
`int(mass)` is probably good enough for the guessing going on here.  It would be nice to get the `O(log(N))` dict lookup rather than looping through the dict.